### ISSUE TITLE
Error in /etc/facter/facts.d/confluence_facts.sh - [[: not found

### ISSUE THREAD

**Issue by mookie-** [voxpupuli/puppet-confluence#82]
### Affected Puppet, Ruby, OS and module versions/distributions
- Puppet: all
- Ruby: all
- Distribution: Ubuntu 14.04
- Module version: 2.2.2
### How to reproduce (e.g Puppet code you use)

Do a puppet run
### What are you seeing

```
>puppet agent --test --noop
/etc/facter/facts.d/confluence_facts.sh: 13: /etc/facter/facts.d/confluence_facts.sh: [[: not found
Info: Retrieving pluginfacts
Info: Retrieving plugin
Info: Loading facts
/etc/facter/facts.d/confluence_facts.sh: 13: /etc/facter/facts.d/confluence_facts.sh: [[: not found
Info: Caching catalog for confluence.example.com
Info: Applying configuration version '1473250250'
Notice: Finished catalog run in 5.67 seconds
```
### What behaviour did you expect instead

No bash errors ;)
### Any additional information you'd like to impart

[[ is a bash buildin. Either change the shebang to `#!/bin/bash` or use `[` instead of `[[`.

**Comment by bastelfreak** [comment]
Hi @mookie-, thanks for rising up the issue. Are you able to provide a PR as well?

**Comment by dhoppe** [comment]
This has been fixed via #95.

**Comment by mookie-** [comment]
@dhoppe this was already fixed by https://github.com/voxpupuli/puppet-confluence/pull/83
